CMEW/meta/rose-meta.conf

```
@@ -115,22 +115,56 @@ type=integer
 
 [template variables=MODEL_ID]
 compulsory=true
-description=A short name for the model simulation.
+description=A short name for the run to be evaluated.
 help=This name is included in file paths so must only contain letters, numbers,
     =underscores or dashes. Therefore spaces, '#' and other special characters are
     =not allowed.
```

```suggestion
help=Must only contain letters, numbers, underscores or dashes.
```
Resolving pre-existing inconsistency in help text.